code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1459,6 +1459,21 @@
 	. = ..()
 	REMOVE_TRAIT(affected_mob, TRAIT_STABLELIVER, type)
 
+/datum/reagent/medicine/lamphrite
+	name = "Lamphrite"
+	description = "A medication that can stabilize inflammated appendices. Not nearly as effective on bursting appendices, though."
+	color = "#ff3589"
+	self_consuming = FALSE
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/medicine/lamphrite/on_mob_metabolize(mob/living/affected_mob)
+	. = ..()
+	ADD_TRAIT(affected_mob, TRAIT_STABLEAPPENDIX, type)
+
+/datum/reagent/medicine/lamphrite/on_mob_end_metabolize(mob/living/affected_mob)
+	. = ..()
+	REMOVE_TRAIT(affected_mob, TRAIT_STABLEAPPENDIX, type)
```

Comment:
I thought stabilizing agent was supposed to have this trait, not some new chem?

Reply:
You misunderstood, read the post again